dev-duckies-sm-proj/src/pages/login/login.css

```
@@ -0,0 +1,19 @@
+body {
+    background-color: #00d1b2; 
+    min-height: 110vh; 
```

`vh` represents a percentage of the height of the viewport's initial containing block. `1vh` is 1% of the viewport height. `100vh` is this max for viewport height.
```suggestion
    min-height: 100vh; 
```